app/bundles/EmailBundle/Views/Email/list.html.php

```diff
@@ -172,6 +172,7 @@
                                 'mautic.email.stat.readcount',
                                 ['%count%' => $item->getReadCount(true)]
                             ); ?></span>
+			<span class="mt-xs label label-primary"><?php echo $view['translator']->trans('mautic.email.stat.readpercent', array('%count%' => round($item->getSentCount(true) > 0 ? (($item->getReadCount(true)/($item->getSentCount(true)))*100) : 0, 2))); ?></span>
```

Comment:
I think this is a good idea!

This line is quite long and hard to read. Do you think you could add a new method instead somewhere here: https://github.com/mautic/mautic/blob/staging/app/bundles/EmailBundle/Entity/Email.php#L596

I have in mind something like
```
    /**
     * @return integer
     */
    public function getReadPercentage($includeVariants = false)
    {
        // The calculation here
    }
```
And then you'd call here just
```
echo $view['translator']->trans('mautic.email.stat.readpercent', array('%count%' => $item->getReadPercentage(true);
```